api/rpc/account/account.go

```diff
@@ -121,12 +120,13 @@ func (s *Server) SignUp(ctx context.Context, in *SignUpRequest) (*empty.Empty, e
 // Verify implements account.Verify
 func (s *Server) Verify(ctx context.Context, in *VerificationRequest) (*empty.Empty, error) {
 	// Validate the token
+	log.Printf("verify token=%s\n", in.Token)
 	claims, err := s.Tokens.ValidateToken(in.Token, auth.TokenTypeVerification)
 	if err != nil {
 		return nil, accounts.InvalidToken
 	}
-	if err := s.Accounts.VerifyUser(ctx, claims.AccountName); err != nil {
-		return nil, convertError(err)
+	if errv := s.Accounts.VerifyUser(ctx, claims.AccountName); err != nil {
```

Comment:
You don't need to rename this error, it's scoped. Plus the test has not been changed.